src/features/movie/components/movieHeader/MovieHeader.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { MovieHeader } from "./MovieHeader";
+
+const movie = {
+  id: "1",
+  title: "Iron Man",
+  year: "05/02/2008",
+  country: "(US)",
+  genre: ["Action", "Science Fiction", "Adventure"],
+  time: "2h 6m",
+  image: "image...(wait for api)",
+  description:
+    "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
+  actors: [
+    { name: "Don Heck", profession: "Characters" },
+    { name: "Jack Kirby", profession: "Characters" },
+    { name: "Jon Favreau", profession: "Director" },
+    { name: "Don Heck", profession: "Screenplay" },
+    { name: "Jack Marcum", profession: "Screenplay" },
+    { name: "Matt Holloway", profession: "Screenplay" },
+  ],
+};
+
+test("movie header", () => {
```

Comment:
It's good to see that tests are being added, but review the best value you can get out of them. This test is not named, nor does it test a lot of assumptions. This might be a good read for you: https://infinum.com/handbook/frontend/react/testing-best-practices

Reply:
Agreed, as soon as I read the documentation I will find a better solution